dags/operators/gcp_container_operator.py

```diff
@@ -78,3 +82,46 @@ def _set_env_from_extras(self, extras):
             os.environ[G_APP_CRED] = service_key.name
             os.environ[GCLOUD_APP_CRED] = service_key.name
             service_key.close()
+
+class GKEClusterCreateOperator(GKEClusterCreateOperator):
+    """
+    We override methods to fix AIRFLOW-4518
```

Comment:
Can you explain in what way we're overriding these? A link to the actual issue instead of its tag would also be helpful, since my initial search for "AIRFLOW-4518" in two search engines did not elucidate.